airflow/www/static/js/graph.js

```diff
@@ -155,8 +156,8 @@ function draw() {
       if (nodeId in taskInstances) tryNumber = taskInstances[nodeId].tryNumber;
       else tryNumber = 0;
 
-      if (task.task_type === 'SubDagOperator') call_modal(nodeId, executionDate, task.extra_links, tryNumber, true);
-      else call_modal(nodeId, executionDate, task.extra_links, tryNumber, undefined);
+      if (task.task_type === 'SubDagOperator') callModal(nodeId, executionDate, task.extra_links, tryNumber, true);
+      else callModal(nodeId, executionDate, task.extra_links, tryNumber, undefined);
     } else {
       // join node between TaskGroup. Ignore.
     }
```

Comment:
```suggestion
    }
```

Reply:
Yes, but this wasn't changed in this PR.